src/docs/markdown/install.md

```diff
@@ -211,3 +212,50 @@ _Note: This is a community-maintained installation method._
 <pre><code class="cmd">scoop install caddy</code></pre>
 
 [**View the Scoop manifest**](https://github.com/ScoopInstaller/Main/blob/master/bucket/caddy.json)
+
+
+## Windows service
```

Comment:
```suggestion
## WinSW service
```

Since this uses WinSW instead of native Windows, right?

Reply:
WinSW is a tool to spin up a _Windows service_. Services are a concept innate to Windows.

Reply:
The alternative (to not require some third party tool) is to write and give a powershell script to do the job, but that's very not fun to maintain or recommend to use. I rather recommend some third party tool which gives it a nicer API (like NSSM or WinSW, TIL of WinSW)

Reply:
How do you do it without a third party tool? (Not that I'm against documenting WinSW, but for official install instructions we should use native facilities.)

Reply:
Software running as a Windows service must adhere to some interfaces, WinSW or nssm are basically wrappers to implement these features and map them to more "standard"/UNIXy process control. WinSW was born in the Jenkins project (IIRC), because implementing those service interfaces in Java isn't that easy (maybe even impossible? I think some features need to be inside the exe file that is registered as the service?)

Reply:
Maybe just support the service interface directly? I just found https://github.com/kardianos/minwinsvc which promises Windows service support with no code changes :+1: